libs/sdk-ui-ext/src/internal/utils/controlsHelper.ts

```diff
@@ -140,3 +149,14 @@ export function minInputValidateAndPushData(
 
     pushData({ properties, propertiesMeta });
 }
+
+export function isSetColumnHeadersPositionToLeftAllowed(insight: IInsightDefinition) {
+    const rowAttribute = bucketsFind(insightBuckets(insight), BucketNames.ATTRIBUTE);
+    const hasRows = rowAttribute && bucketItems(rowAttribute).length > 0;
+    const columnAttribute = bucketsFind(insightBuckets(insight), BucketNames.COLUMNS);
```

Comment:
please rename to columnAttribute -> columnBucket

Reply:
fixed